CompetitionTemplate/tests/competition/subsystems/low_bar_to_low_goal/DriveToTurningPointTest.java

```diff
@@ -0,0 +1,52 @@
+package competition.subsystems.low_bar_to_low_goal;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import competition.subsystems.autonomous.DriveForDistanceCommand;
+import competition.subsystems.autonomous.LowBarScoreCommandGroup;
+import competition.subsystems.drive.DriveTestBase;
+import competition.subsystems.drive.PoseSubsystem;
+import edu.wpi.first.wpilibj.MockEncoder;
+
+public class DriveToTurningPointTest extends DriveTestBase{
```

Comment:
I think we could probably get rid of this test case, it doesn't really test anything that the DriveForDistanceCommandTest doesn't already test?